src/commands/backend/list.ts

```diff
@@ -37,7 +37,12 @@ export default class List extends Command {
             '-'
         },
       },
-      lock: { header: 'Lock' },
+      lock: {
+        header: 'Lock',
+        get: row => {
+          return row.lock ? row.lock : ''
+        },
```

Comment:
nit: could slim this down to:

```ts
get: row => row.lock ?? ''
```

What you've got works too, though, if you prefer it!